index.bs

```diff
@@ -74,20 +67,88 @@ urlPrefix: https://tools.ietf.org/html/draft-ietf-httpbis-header-structure; spec
     "href": "https://channel9.msdn.com/Events/WebPlatformSummit/2015/The-Microsoft-Edge-Rendering-Engine-that-makes-the-Web-just-work#time=9m45s",
     "title": "The Microsoft Edge Rendering Engine that makes the Web just work",
     "author": [ "Jacob Rossi" ]
+  },
+  "51Degrees": {
+    "href": "https://51degrees.com/device-detection",
+    "title": "Use cases for device detection with 51Degrees",
+    "author": [ "51Degrees" ]
+  },
+  "Scientiamobile": {
+    "href": "https://www.scientiamobile.com",
+    "title": "WURFL based device detection",
+    "author": [ "Scientiamobile" ]
+  },
+  "DeviceAtlas": {
+    "href": "https://deviceatlas.com/",
+    "title": "Use cases for device detection with DeviceAtlas",
+    "author": [ "DeviceAtlas" ]
+  },
+  "AlexRussell": {
+    "href": "https://twitter.com/slightlylate/status/1139684093602349056/photo/2",
+    "title": "Performance scores by device model",
+    "author": [ "Alex Russell" ]
+  },
+  "Google-Analytics": {
+    "href": "https://support.google.com/analytics/answer/3123672?hl=en",
+    "title": "Acquisition device",
+    "author": [ "Google" ]
+  },
+  "Client-Hints-Infrastructure": {
+    "href": "https://wicg.github.io/client-hints-infrastructure/",
+    "title": "Client Hints Infrastructure",
+    "author": [ "Yoav Weiss" ]
+  },
+  "CMA-UK": {
+    "href": "https://www.gov.uk/cma-cases/online-platforms-and-digital-advertising-market-study",
+    "title": "Online platforms and digital advertising market study",
+    "author": [ "UK Competition and Markets Authority" ]
+  },
+  "Privacy-Budget": {
+    "href": "https://github.com/bslassey/privacy-budget",
+    "title": "Combating Fingerprinting with a Privacy Budget",
+    "author": [ "Brad Lassey" ]
   }
 }
 </pre>
 
+Dependencies {#dependencies}
+============
+
+This document is dependent on the following related specifications which the reader must be 
+familiar with before reading this document.
+
+1.  Experimental IETF specification for Client Hints ([[I-D.draft-ietf-httpbis-client-hints]]) 
+    which expires in January 2021.
+    
```

Comment:
That's misleading. The draft has been approved by the IESG and is in the RFC Publication Queue.

Reply:
@reschke could you provide some links and more information? The edit was based on the status of the document referenced originally.

Reply:
Links: <https://datatracker.ietf.org/doc/draft-ietf-httpbis-client-hints/> and <https://www.rfc-editor.org/current_queue.php#draft-ietf-httpbis-client-hints>

The *draft* itself does indeed expire 6 months after publication, but that really is irrelevant; in particular because it will be published as RFC before that time.

Reply:
I've reviewed the [RFC2026](https://tools.ietf.org/html/rfc2026#section-4.2.1), and its amendments, and do not believe this dependent specification is on any standards track within the IETF. The definition of Experimental status is as follows.

>  The "Experimental" designation typically denotes a specification that
   is part of some research or development effort.  Such a specification
   is published for the general information of the Internet technical
   community and as an archival record of the work, subject only to
   editorial considerations and to verification that there has been
   adequate coordination with the standards process (see below).  An
   Experimental specification may be the output of an organized Internet
   research effort (e.g., a Research Group of the IRTF), an IETF Working
   Group, or it may be an individual contribution.

The pull request has been amended to reference RFC2026 and highlight the experimental nature of the Client Hints document.

Reply:
ack